blades/dim_blade.h

```diff
@@ -1,13 +1,24 @@
 #ifndef BLADES_DIM_BLADE_WRAPPER_H
 #define BLADES_DIM_BLADE_WRAPPER_H
 
-class DimBladeWrapper : public BladeWrapper, BladeStyle {
+class DimBladeWrapper : public BladeWrapper, BladeStyle 
+#ifdef ENABLE_MENU_DIMBLADE
+      ,protected SaberBase
+#endif
+      {
 public:
   DimBladeWrapper(BladeBase* blade, int fraction) {
     blade_ = blade;
+#ifdef ENABLE_MENU_DIMBLADE
+    SaberBase::SetBrightness((float)fraction/16384.0*100.0);
```

Comment:
Actually, I think this works really weird.
Basically, it seems like the original setting of DimBlade is completely ignored.
It would make more sense to me if the brightness setting from SaberBase and the argument were multiplied.

Reply:
No, The setting value in your config.h file DimBlade(##.#, ...  will actually be your initial brightness for your blade. (with or without the ENABLE_MENU_DIMBLADE
It also mean that you don't have to initialize it in ProffieOS.ino, 

SaberBase::DimChangeMode SaberBase::dim_change_mode_ =
  SaberBase::DIM_CHANGE_MODE_NONE;
uint32_t SaberBase::current_brightness_; // does not need to be initialized, takes setting from config.h